File: ProgrammersLifeiOS/ProgrammersLifeiOS/PointStats.m
//
//  PointStats.m
//  ProgrammersLifeiOS
//
//  Created by Zackary T Misso on 7/6/15.
//  Copyright © 2015 Zackary Misso. All rights reserved.
//

#import "PointStats.h"

@implementation PointStats

-(id)init{
    if(self = [super init]){
        // to be implemented
    }
    return self;
}

// getter methods
-(int)getBrainPoints { return brainPoints; }
-(int)checkPoints { return checkPoints; }
-(int)starPoints { return starPoints; }

// setter methods
-(void)setBrainPoints:(int)value { brainPoints = value; }
-(void)setCheckPoints:(int)value { checkPoints = value; }
-(void)setStarPoints:(int)value { starPoints = value; }

@end

File: ProgrammersLifeDemo/src/player/PointStats.java
// This is the Game Tester's version of ProgrammersLife
// Author :: Zackary Misso / FiralStudios
// Version :: 0.0.1

package player;

public class PointStats {
	private int brainPoints;
	private int checkPoints;
	private int starPoints;
	
	public void initializeNewGame(){
		brainPoints = 0;
		checkPoints = 0;
		starPoints = 0;
	}
	
	public String toString(){
		// to be implemented
		return "";
	}
	
	// getter methods
	public int getBrainPoints(){return brainPoints;}
	public int getCheckPoints(){return checkPoints;}
	public int getStarPoints(){return starPoints;}
	
	// setter methods
	public void setBrainPoints(int param){brainPoints = param;}
	public void setCheckPoints(int param){checkPoints = param;}
	public void setStarPoints(int param){starPoints = param;}
}
